Models/Text/BytePairEncoding.swift

```diff
@@ -82,13 +133,26 @@ extension BytePairEncoder {
     }
 
     internal static let defaultGlossary: [String] = [
-        "e.g", "i.e", "&amp;", "&#124;", "&lt;", "&gt;", "&apos;", "&quot;", "&#91;", "&#93;"]
+        "e.g", "i.e", "&amp;", "&#124;", "&lt;", "&gt;", "&apos;", "&quot;", "&#91;", "&#93;",
+    ]
 
     internal static let defaultGlossaryRegex: NSRegularExpression = {
         let escapedGlossary = defaultGlossary.map { "\\Q\($0)\\E" }.joined(separator: "|")
         return try! NSRegularExpression(pattern: "(?:\(escapedGlossary))|(?!\(escapedGlossary))")
     }()
 
+    /// Regular expression matching the OpenAI GPT-2 implementation.
+    internal static let gpt2Glossary: [String] = [
```

Comment:
```suggestion
    internal static let gpt2Glossary = [
```

Does the type not get inferred automatically?